mavros/src/plugins/setpoint_position.cpp

```diff
@@ -181,9 +185,43 @@ class SetpointPositionPlugin : public plugin::PluginBase,
 	}
 
 	/**
-	 * Gets gps setpoint, converts it to local ENU, and sends it to FCU
+	 * Gets setpoint position setpoint and send SET_POSITION_TARGET_GLOBAL_INT
 	 */
 	void setpointg_cb(const mavros_msgs::GlobalPositionTarget::ConstPtr &req)
+	{
```

Comment:
I do think this should be limited to ROS std msgs instead of `mavros_msgs`. The main reason is that using this wouldn't be different from the `setpoint_raw` plugin. For the `setpoint_position`, you only send pose (poisition+orientation) setpoints. That's why [`geographic_msgs/GeoPoseStamped`](http://docs.ros.org/jade/api/geographic_msgs/html/msg/GeoPoseStamped.html) makes sense, similar to the usage of `geometry_msgs/PoseStamped` in `setpoint_cb`.

Reply:
Thanks, I think it makes sense to have it subscribe to `geographic_msgs` and simplify the interface compared to the set-it-all-yourself `setpoint_raw` messages